pkg/make_deb.py

```diff
@@ -386,4 +400,5 @@ def main(unused_argv):
   # https://docs.python.org/3/library/sys.html#sys.argv
   # https://docs.python.org/3/library/os.html#os.fsencode
   # https://www.python.org/dev/peps/pep-0383/
-  main(FLAGS([os.fsencode(arg).decode('utf-8') for arg in sys.argv]))
+  # main(FLAGS([os.fsencode(arg).decode('utf-8') for arg in sys.argv]))
```

Comment:
Stale line. Also should the above comment be deleted now that the logic's moved to `GetFlagValue`?